timetable/extractor.py

```diff
@@ -73,60 +73,43 @@ def build_courses(html: str, course_names: dict) -> list[Course]:
 
     timings = create_timings(table)
     rows = table.find_all('tr')
+    
     for row in rows:
         day_cell: PageElement = row.find('td', {'valign': 'top'})
         if not day_cell:
             continue
-        day = DAYS_MAP.get(day_cell.get_text(), day_cell.get_text())
-
         # This is the previously parsed course/time slot. Used to merge timeslots occurring adjacent to each other and initialize course objects
+        day = DAYS_MAP.get(day_cell.get_text(), day_cell.get_text())
+    
         prev: Course | None = None
         cells = [cell for cell in row.find_all('td') if cell.attrs.get('valign') != 'top']
-
+    
         for index, cell in enumerate(cells):
-            # Empty cell means either previous class just ended or no class is scheduled
-            if cell.get_text().strip() == "":
+            text = cell.get_text().strip()
+            if not text or text in [' ', ' ']: # encountered empty cell: either commit the prev course or skip
```

Comment:
The `.strip()` method seems to remove those two special characters as well. Can the statement be repaced with just `if not text`?

Reply:
alright